docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.8'
+
+services:
+  postgres:
+    image: postgres:14-alpine
+    # comment out if you want to externally connect DB
+    # ports:
+    #   - 5432:5432
+    volumes:
+      - ./postgres-data:/var/lib/postgresql/data
+    environment:
+      - PGDATA=/var/lib/postgresql/data/pgdata
+      - POSTGRES_USER=skyvern
+      - POSTGRES_PASSWORD=skyvern
+      - POSTGRES_POSTGRES_DB=skyvern
+
+  skyvern:
+    image: public.ecr.aws/t6d4b5t4/skyvern:latest
+    restart: on-failure
+    # comment out if you want to externally call skyvern API
+    # ports:
+    #   - 8000:8000
+    volumes:
+      - ./artifacts:/data/artifacts
+      - ./videos:/data/videos
+      - ./har:/data/har
+      - ./.streamlit:/app/.streamlit
+    environment:
```

Comment:
can we point it to the `.env` file?

Reply:
If `.env` pointed, users need to run the docker-compose together with `.env`.
In this way, rookie users could run the single `docker-compose.yml` file.
And advanced users could point it to the `.env` by themselves.

Reply:
Besides, if we change the settings in env, we are supposed to restart/recreate the container.
If we have an isolated env setting file, I think it's easy to forget to restart/recreate.
I think load from `.env` could be more useful if we want to share the same env between multiple containers, rather than running a single container